vm_insnhelper.c

```diff
@@ -3502,18 +3502,18 @@ vm_invoke_block(rb_execution_context_t *ec, rb_control_frame_t *reg_cfp,
                 struct rb_calling_info *calling, const struct rb_callinfo *ci,
                 bool is_lambda, VALUE block_handler)
 {
+    auto VALUE (*func)(rb_execution_context_t *ec, rb_control_frame_t *reg_cfp,
```

Comment:
Does this `auto` have any sense?

Reply:
Good catch.  I thought functions neither `static` nor `extern` shall explicit be marked `auto`.  (That was true, [here you are the reason](https://gcc.gnu.org/onlinedocs/gcc/Nested-Functions.html#Nested-Functions),) but not the case for function _pointers_.  This was my confusing them.